editor/inspector/editor_resource_picker.cpp

```diff
@@ -107,19 +108,32 @@ void EditorResourcePicker::_update_resource() {
 
 			String tooltip;
 
+			String resource_name = "resource";
+			if (edited_resource.is_valid()) {
+				resource_name = edited_resource->get_class();
+
+				Ref<Script> custom_script = PropertyUtils::get_custom_type_script(*edited_resource);
+				if (custom_script.is_valid()) {
+					String global_name = custom_script->get_global_name();
+					if (!global_name.is_empty()) {
+						resource_name = global_name;
+					}
+				}
+			}
+
 			if (num_of_copies > 1) {
-				tooltip = vformat(TTR("This Resource is used in (%d) places."), num_of_copies);
+				tooltip = vformat(TTRN("This %s is used in %d place.", "This %s is used in %d places.", num_of_copies), resource_name, num_of_copies);
 			} else if (!is_internal) {
-				tooltip = TTR("This Resource is external to scene.");
+				tooltip = vformat(TTR("This %s is external to scene."), resource_name);
 			}
 
 			if (!editable) {
-				tooltip += "\n" + TTR("The Resource cannot be edited in the inspector and can't be made unique directly.") + "\n";
+				tooltip += "\n" + vformat(TTR("The %s cannot be edited in the inspector and can't be made unique directly."), resource_name) + "\n";
 			} else {
-				tooltip += unique_enable ? TTR(" Left-click to make it unique.") + "\n" : "\n";
+				tooltip += "\n" + TTR("Left-click to make it unique.") + "\n";
```

Comment:
What happened to `unique_enable`?
It should be something like
```suggestion
				if (unique_enable) {
					tooltip += "\n" + TTR("Left-click to make it unique.") + "\n";
				}
```

Reply:
Nice catch, I misread the logic there and removed the unique_enable by mistake.